pennylane/transforms/__init__.py

```diff
@@ -44,15 +44,26 @@
     :toctree: api
 
     ~adjoint
+    ~ctrl
+    ~transforms.invisible
+    ~apply_controlled_Q
+    ~quantum_monte_carlo
+
+Transforms for circuit compilation and optimization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This set of transforms accept quantum functions, and perform basic circuit
+optimization tasks.
```

Comment:
```suggestion
Transforms for circuit compilation
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

This set of transforms accept quantum functions, and perform basic circuit compilation tasks.
```

I was reading through the rendered docs, and realized this might be a bit confusing due to usage of 'optimization' elsewhere in the docs 🤔 It's quite an overloaded term in PL!

Reply:
Oh yeah, good point!